google/cloud/pubsub/samples/samples.cc

```diff
@@ -780,30 +785,15 @@ void CreateAvroSchema(google::cloud::pubsub::SchemaServiceClient client,
   //! [START pubsub_create_avro_schema] [create-avro-schema]
   namespace pubsub = ::google::cloud::pubsub;
   [](pubsub::SchemaServiceClient client, std::string const& project_id,
-     std::string const& schema_id) {
-    auto constexpr kDefinition = R"js({
-      "type": "record",
-      "name": "State",
-      "namespace": "utilities",
-      "doc": "A list of states in the United States of America.",
-      "fields": [
-        {
-          "name": "name",
-          "type": "string",
-          "doc": "The common name of the state."
-        },
-        {
-          "name": "post_abbr",
-          "type": "string",
-          "doc": "The postal code abbreviation of the state."
-        }
-      ]
-    })js";
+     std::string const& schema_id, std::string const& schema_definition_file) {
+    auto definition = ReadFile(schema_definition_file);
+    if (!definition) throw std::move(definition).status();
+
     google::pubsub::v1::CreateSchemaRequest request;
     request.set_parent(google::cloud::Project(project_id).FullName());
     request.set_schema_id(schema_id);
     request.mutable_schema()->set_type(google::pubsub::v1::Schema::AVRO);
-    request.mutable_schema()->set_definition(kDefinition);
+    request.mutable_schema()->set_definition(definition.value());
```

Comment:
Once we check a value for a `StatusOr<>` we prefer (and recommend) `*definition`.